release/release_cmds.sh

```diff
@@ -100,6 +101,23 @@ publish_candidate() {
   cd $BASE_DIR
 }
 
+release_audit() {
+  # Downloads/runs the apache Release Audit Tool on the base folder, excluding this
+  # release directory
+  #
+  if [ ! -d apache-rat-0.12 ] ; then
+    curl http://apache.spinellicreations.com//creadur/apache-rat-0.12/apache-rat-0.12-bin.tar.bz2 > rat.tar.bz2
+    tar -xzf rat.tar.bz2
+    rm rat.tar.bz2
+  fi
+  java -jar apache-rat-0.12/apache-rat-0.12.jar -e release -e build -e cmake* -d .. > rat-report.txt
+  if [ $? == 0 ] ; then
+     echo "created rat-report.txt"
+  else
+     echo "error running rat"
+  fi
```

Comment:
It's a jar - like a java executable. I think it's fine to keep around seeing as gitignore already ignores it.